datashader/glyphs.py

```diff
@@ -66,3 +67,182 @@ def _compute_x_bounds(self, df):
 
     def _compute_y_bounds(self, df):
         return df[self.y].min(), df[self.y].max()
+
+
+class Line(Glyph):
+    """A point, with points defined by ``x`` and ``y``.
+
```

Comment:
Ok.  Matching bokeh is a good enough reason.  Still needs updated docstring, though.